code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -84,6 +84,8 @@ GLOBAL_LIST_INIT(common_loot, list( //common: basic items
 		/obj/item/weldingtool = 1,
 		/obj/item/wirecutters = 1,
 		/obj/item/wrench = 1,
+		/obj/item/gun/ballistic/automatic/pistol/no_mag = 1,
```

Comment:
```suggestion

```
Guns should not be in the common loot pool, also this weapon should not be findable yet